LongWarOfTheChosen/Src/WotCWeaponsAndArmor/Classes/X2Item_Armors.uc

```diff
@@ -0,0 +1,265 @@
+
+	// Cost
+    Resources.ItemTemplateName = 'Supplies';
+    Resources.Quantity = 30;
+    Template.Cost.ResourceCosts.AddItem(Resources);
+
+    Resources.ItemTemplateName = 'AlienAlloy';
+    Resources.Quantity = 5;
+    Template.Cost.ResourceCosts.AddItem(Resources);
+
+	return Template;
+}
+
+static function X2DataTemplate CreatePoweredSkirmisherArmor()
+{
+	local X2ArmorTemplate Template;
+	local ArtifactCost Resources;
+
+	`CREATE_X2TEMPLATE(class'X2ArmorTemplate', Template, 'PoweredSkirmisherArmor');
+	Template.strImage = "img:///UILibrary_XPACK_StrategyImages.Inv_HORArmorPowr";
+	Template.ItemCat = 'armor';
+	Template.bAddsUtilitySlot = false;
+	Template.StartingItem = false;
+	Template.CanBeBuilt = true;
+	Template.bInfiniteItem = false;
+	Template.TradingPostValue = 60;
+	Template.PointsToComplete = 0;
+	Template.Abilities.AddItem('MediumPoweredArmorStats');
+	Template.ArmorTechCat = 'powered';
+	Template.ArmorCat = 'skirmisher';
+	Template.Tier = 3;
+	Template.AkAudioSoldierArmorSwitch = 'Warden';
+	Template.EquipNarrative = "X2NarrativeMoments.Strategy.CIN_ArmorIntro_PoweredMedium";
+	Template.EquipSound = "StrategyUI_Armor_Equip_Powered";
+
+	Template.Requirements.RequiredTechs.AddItem('PoweredArmor');
+
+	Template.SetUIStatMarkup(class'XLocalizedData'.default.HealthLabel, eStat_HP, class'X2Ability_ItemGrantedAbilitySet'.default.MEDIUM_POWERED_HEALTH_BONUS, true);
+	Template.SetUIStatMarkup(class'XLocalizedData'.default.ArmorLabel, eStat_ArmorMitigation, class'X2Ability_ItemGrantedAbilitySet'.default.MEDIUM_POWERED_MITIGATION_AMOUNT);
+
+	// Cost
+    Resources.ItemTemplateName = 'Supplies';
+    Resources.Quantity = 80;
+    Template.Cost.ResourceCosts.AddItem(Resources);
+
+    Resources.ItemTemplateName = 'AlienAlloy';
+    Resources.Quantity = 10;
+    Template.Cost.ResourceCosts.AddItem(Resources);
+
+    Resources.ItemTemplateName = 'EleriumDust';
+    Resources.Quantity = 10;
+    Template.Cost.ResourceCosts.AddItem(Resources);
+
+	return Template;
+}
+
+static function X2DataTemplate CreatePlatedTemplarArmor()
+{
+	local X2ArmorTemplate Template;
+	local ArtifactCost Artifacts;
+	local ArtifactCost Resources;
+
+	`CREATE_X2TEMPLATE(class'X2ArmorTemplate', Template, 'PlatedTemplarArmor');
+	Template.strImage = "img:///UILibrary_XPACK_StrategyImages.Inv_HORArmorPlat";
+	Template.ItemCat = 'armor';
+	Template.bAddsUtilitySlot = false;
+	Template.StartingItem = false;
+	Template.CanBeBuilt = true;
+	Template.bInfiniteItem = false;
+	Template.TradingPostValue = 20;
+	Template.PointsToComplete = 0;
+	Template.Abilities.AddItem('MediumPlatedArmorStats');
+	Template.ArmorTechCat = 'plated';
+	Template.ArmorCat = 'templar';
+	Template.Tier = 1;
+	Template.AkAudioSoldierArmorSwitch = 'Predator';
+	Template.EquipNarrative = "X2NarrativeMoments.Strategy.CIN_ArmorIntro_PlatedMedium";
+	Template.EquipSound = "StrategyUI_Armor_Equip_Plated";
+
+    Template.Requirements.RequiredTechs.AddItem('PlatedArmor');

```

Comment:
Bonus points if you manage to make indent consistent, i.e. tabs instead of spaces. Note that the LW2 code base is a mess when it comes to whitespace, so this would fit right in :smile: Would prefer consistency though.

Reply:
Arrrgghh I am a spaces guy myself :P. I'll make sure to get the indenting fixed here.

Reply:
Trust me, I prefer spaces too, but the convention for highlander and most of LW2 is tabs. When in Rome...

Reply:
All of the files should now be using tabs instead of spaces for indentation.